### PR TITLE
doc: update security advisory for 1.6.1 release

### PR DESCRIPTION
**Pull request by yonghuah**
Update mitigations for security vulnerabilities
 for ACRN 1.6.1 release

Signed-off-by: Yonghua Huang <yonghua.huang@intel.com>

### CODE REVIEW

**Comment by yonghuah** [comment]
@NanlinXie  please review asa update for 1.6.1 release , thanks.

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/asa.rst [hunk 1]
```diff
@@ -3,6 +3,22 @@
 Security Advisory
 #################
 
+Addressed in ACRN v1.6.1
+************************
+
+We recommend that all developers upgrade to this v1.6.1 release (or later), which
+addresses the following security issue that were discovered in previous releases:
```
Comment (dbkinder) [inline doc/asa.rst:10]: Change ``were`` to ``was``

Comment (dbkinder) [reply doc/asa.rst]: to expedite things, I made this change directly to @yonghuah branch since it was a simple grammar fix

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by dbkinder** [comment]
@NanlinXie I'm going to merge this now.  You can submit an update if needed.

**Pull request merged**